### PR TITLE
Fixes #1 Running memcached with options

### LINKED ISSUE #1: CMD in Dockerfile
Are there any plans to include the following CMD (switches for memcached) in the Dockerfile?

E.g. -m <memory_to-use> -vv

CACHE_SIZE appears to be similar to the -m switch, but I think that may have been for older releases of memcached.

-vv is for printing out errors and warnings, client commands and responses

Reason I mention this is that I see that FeedHenry and 3scale use these switches.

### PR DESCRIPTION
**Pull request by phracek**
This Pull Request adds several options into memcached container like
* cache_size
* connections
* threads

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,20 @@ Run the container
 docker run -it -p 11211:11211 --name memcached modularitycontainers/memcached
 ```
 
-If you would like to increase a CACHE_SIZE use environment variable -e CACHE_SIZE:
+If you would like to debug memcached, use container option =e MEMCACHED_DEBUG_MODE:
```
Comment (TomasTomecek) [inline README.md]: `s/=/-/`

File: README.md [hunk 2]
```diff
@@ -16,9 +16,20 @@ Run the container
...
 ```bash
-docker run -it -e CACHE_SIZE=128 -p 11211:11211 --name memcached modularitycontainers/memcached
+docker run -it -p 11211:11211
```
Comment (TomasTomecek) [inline README.md]: I just tried this and am getting zero output: so how is this option helpful in debugging?

```
$ docker run --rm -ti -e MEMCACHED_DEBUG_MODE modularitycontainers/memcached
```

File: README.md [hunk 3]
```diff
@@ -16,9 +16,20 @@ Run the container
...
+[-e MEMCACHED_DEBUG_MODE]
+--name memcached modularitycontainers/memcached
+```
+
+If you would like to change memcached options, like cache_size, connections or threads, use environment variable -e MEMCACHED_CACHE_SIZE, -e MEMCACHED_CONNECTIONS, -e MEMCACHED_THREADS respectively:
+```bash
+docker run -it -p 11211:11211
```
Comment (TomasTomecek) [inline README.md]: what am I doing wrong?

```
$ docker run --rm -ti -e MEMCACHED_CACHE_SIZE=111 -e MEMCACHED_CONNECTIONS=25 -e MEMCACHED_THREADS=32 modularitycontainers/memcached
[err] evsig_init: socketpair: Too many open files
```

**Review by TomasTomecek — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (phracek) [reply README.md]: You have missed -p 11211:11211. But nevertheless, I have to find out proper way how to debug memcached.

File: README.md (hunk 3 above)
Comment (phracek) [reply README.md]: Again missing -p 11211:11211. But too many open files. It seems like the number is a bit small. https://github.com/memcached/memcached/wiki/ConfiguringServer#connection-limit

The number 250 works fine.

Comment (TomasTomecek) [reply README.md]: So I would suggest documenting what the defaults are so people are not confused as I was.

**Comment by TomasTomecek** [comment]
I would suggest documenting the new variables in the help file. I really like [this table](https://github.com/sclorg/postgresql-container/tree/master/9.5) sclorg is using to document env vars.

**Comment by phracek** [comment]
Thanks. That's a really good point.

**Review by TomasTomecek — APPROVED** [review]
LGTM, nicely done!

**Pull request merged**